ifacemonitor/ifacemonitor_test.go

```diff
@@ -351,12 +370,29 @@ var _ = Describe("ifacemonitor", func() {
 	})
 
 	It("should skip netlink address updates for ipvs", func() {
-		SetDefaultConsistentlyDuration(1 * time.Second)
 		nl.addLink("kube-ipvs0")
 		resyncC <- time.Time{}
 		dp.expectAddrStateCb("kube-ipvs0", "", true)
 		nl.addAddr("kube-ipvs0", "10.100.0.1/32")
+		// Should not expect an address notification from interface monitor to dataplane.
 		dp.notExpectAddrStateCb()
+
+		// Should not expect a link up/down/delete if link has been added but excluded for moinitoring.
+		nl.changeLinkState("kube-ipvs0", "up")
+		dp.notExpectLinkStateCb()
+		nl.changeLinkState("kube-ipvs0", "down")
+		nl.delLink("kube-ipvs0")
+		dp.notExpectLinkStateCb()
```

Comment:
Is there an AddrStateCb here?  It feels asymmetric if there is a callback when the link is created, but not when it is deleted.  Could we either:
- ensure that there is a symmetric callback when the link is deleted, or
- suppress the callback when the link is created?

Reply:
(I think probably the latter would be preferable.)